opensearch_py_ml/_version.py

```diff
@@ -27,7 +27,7 @@
 __title__ = "opensearch_py_ml"
 __description__ = "Python Client and Toolkit for DataFrames, Big Data, Machine Learning and ETL in OpenSearch"
 __url__ = "https://github.com/opensearch-project/opensearch-py-ml"
-__version__ = "1.0.0-beta1"
+__version__ = "1.0.0-b1"
```

Comment:
Not sure if this is the best practice to do so. 
@dblock Any suggestion or input here?

Reply:
I would release a 0.1.0, keep iterating on it until we're ready for a 1.0.

Reply:
From the build log, looks like setuptools (the library to build, install the package) auto normalizing this:
```
/private/var/folders/56/2tdnnvpn26n8f2jqp2ntxnyw0000gr/T/build-env-um0_7492/lib/python3.9/site-packages/setuptools/dist.py:530: UserWarning: Normalizing '1.0.0-beta1' to '1.0.0b1'
```
And this is happening due to [PEP 440](https://peps.python.org/pep-0440/#public-version-identifiers)
So PEP 440 allows only five suffixes: a, b, rc, post and dev. `b` for `beta`.

To keep this consistent across everywhere I decided to keep `1.0.0b1`

Reply:
Semver (https://semver.org/) says `-beta.1` is the correct label otherwise

Reply:
In that case 1.0.0-beta1 seems more appropriate as b1 might be confusing for the users

Reply:
@dblock you meant to make the version `0.1.0-beta1`?

Reply:
I believe he meant either `0.1.0` or `-beta1`

Reply:
@dblock @gaiksaya [python](https://www.python.org/downloads/release/python-350b1/) is also following the similar approach : `Python 3.5.0b1`

Reply:
I think we should do what Python does. 

Either way, please ignore me, @gaiksaya just asked for my opinion, but I am not the one to decide what we should do here.

On 3.0.0 beta, _personaly_ I don't believe in beta releases, because they imply the project doesn't have mechanisms to release high quality software often. I'd prefer to confidently release 3.0, and if anyone wants to use an earlier build, not to bless a particular version as "beta".